components/authz-service/server/v2/policy_test.go

```diff
@@ -41,25 +41,49 @@ var dummyWriter engine.V2pXWriter = nil
 func TestCreatePolicy(t *testing.T) {
 	ctx := context.Background()
 	prng := prng.Seed(t)
-	ts := setupV2WithWriter(t, dummyWriter)
+	ts := setupV2p1WithWriter(t, dummyWriter)
```

Comment:
Had to update all the tests to run against 2.1 since the server can no longer start in 2.0!